<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>688</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page"/>
         <widget class="QWidget" name="page_2">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="sizeConstraint">
                <enum>QLayout::SetMinimumSize</enum>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetMaximumSize</enum>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Update Speed</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="updateSpeedSelector">
                   <item>
                    <property name="text">
                     <string>Normal</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Slow</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Realtime</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>No Update</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QCheckBox" name="showAllCheckbox">
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>Show For All Users</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="procTableWidget">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContents</enum>
               </property>
               <property name="autoScroll">
                <bool>true</bool>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::ExtendedSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="showGrid">
                <bool>false</bool>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>100</number>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>100</number>
               </attribute>
               <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderCascadingSectionResizes">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>USER</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>PID</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>CPU%</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>MEM%</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>STATUS</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>COMMAND</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <widget class="QMenu" name="menuUpdate_Speed">
     <property name="title">
      <string>Update Speed</string>
     </property>
     <addaction name="actionUpdateNormal"/>
     <addaction name="actionUpdateSlow"/>
     <addaction name="actionUpdateRealtime"/>
     <addaction name="actionUpdateNoUpdate"/>
    </widget>
    <addaction name="menuSendSignal"/>
    <addaction name="separator"/>
    <addaction name="menuUpdate_Speed"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProcess"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="menuSendSignal">
   <property name="text">
    <string>Send Signal</string>
   </property>
  </action>
  <action name="actionUpdateNormal">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionUpdateSlow">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Slow</string>
   </property>
  </action>
  <action name="actionUpdateRealtime">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Realtime</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUpdateNoUpdate">
   <property name="text">
    <string>No Update</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
